test/bigtest/tests/location-libraries-test.js

```diff
@@ -20,7 +20,7 @@ describe('LocationLibraries', () => {
   describe('Institution is present', () => {
     beforeEach(function () {
       institution = this.server.create('institution');
-      this.visit('/settings/tenant-settings/location-libraries');
+      return this.visit('/settings/tenant-settings/location-libraries');
```

Comment:
Is `return` is really needed?

Reply:
Oh this was my mistake. I thought they were asynchronous and believed that was the cause of the inconsistency but those methods are synchronous. I undid the `return`.